layouts/_default/single.html

```diff
@@ -22,7 +22,7 @@ <h1 id="page-title"><a href="{{ .Site.BaseURL }}">{{ .Site.Data.config.page_titl
           <li><a href="{{ .Permalink }}">{{ .LinkTitle | humanize }}</a></li>
           {{ end }}
       </ul>
-      {{ if (and $.Site.Data.config.enableToc (not (.Params.disableToc))) }}
+      {{ if (and $.Site.Data.config.enableToc (ne .Params.enableToc false)) }}
```

Comment:
Can be simplified I think

```suggestion
      {{ if (and $.Site.Data.config.enableToc .Params.enableToc) }}
```

Reply:
That was my first try, and unfortunately we can't use like that.

Reasoning:
If `enableToc` is absent, it defaults to `false`. Resulting in not rendering the ToC even if it's enabled in `data/config.yaml`.

That's why the logic checks for an explicit `enableToc: false`.

Reply:
(I marked this as "resolved" but I guess it's your call. Feel free to "unresolve" it if you think it needs more discussion.)

Reply:
This makes sense to me! Sorry, was just offline for a bit